accord-core/src/main/java/accord/coordinate/CoordinateTransaction.java

```diff
@@ -84,7 +86,7 @@ void onPreAccepted(Topologies topologies, Timestamp executeAt, List<PreAcceptOk>
             //                                  but by sending accept we rule out hybrid fast-path
             // TODO (low priority, efficiency): if we receive an expired response, perhaps defer to permit at least one other
             //                                  node to respond before invalidating
-            if (executeAt.isRejected() || node.agent().isExpired(txnId, executeAt.hlc()))
+            if (executeAt.isRejected() || executeAt.hlc() - txnId.hlc() >= node.agent().preAcceptTimeout(MICROSECONDS))
```

Comment:
this is true for C* but is this true for accord?  I thought `hlc` was kinda impl specific...

Reply:
`accord.primitives.Timestamp#hlc`

```
    /**
     * A hybrid logical clock with implementation-defined resolution
     */
    public long hlc()
```

Reply:
Good catch

Reply:
Good point indeed. And it's actually trivially fixed and looks a bit simpler once addressed. Pushed and update.